OracleOpenJDK/README.md

```diff
@@ -0,0 +1,18 @@
+Oracle Java OpenJDK
+=====
+This repository contains a sample Docker configuration to facilitate installation and environment setup for DevOps users. This project includes a Dockerfile for Oracle OpenJDK 14 based on Oracle Linux.
+
+## Building the Java 14 (OpenJDK) base image
+[Download OpenJDK 14](https://jdk.java.net/14/) `.tar.gz` file and drop it inside the folder `../OracleOpenJDK/14`.
```

Comment:
As above, this should be done automatically for the user during build. There's no need to require a manual download because the license doesn't need to be manually accepted.